csrc/executor.cpp

```diff
@@ -2016,7 +2018,7 @@ flatbuffers::Offset<serde::CudaKernel> FusionExecutor::serialize(
     uint8_t* ptx_ptr = nullptr;
     fb_ptx =
         builder.CreateUninitializedVector(compiled_kernel.ptx.size(), &ptx_ptr);
-    std::copy(compiled_kernel_.ptx.begin(), compiled_kernel.ptx.end(), ptx_ptr);
+    std::copy(compiled_kernel.ptx.begin(), compiled_kernel.ptx.end(), ptx_ptr);
```

Comment:
Is this a bug in main?

Reply:
Potentially. Luckily, `compiled_kernel` and `compiled_kernel_` are always the same. But this brings up another question whether this particular variation of `serialize` can be a static method instead...